main/src/library/Nel.flix

```diff
@@ -368,9 +368,14 @@ namespace Nel {
     /// That is, the result is of the form: `f(x1, ...lazy f(xn-1, lazy f(xn, s))...)`.
     /// The foldRightLazy function exists to allow early termination of a fold.
     ///
-    pub def foldRightLazy(f: (a, Lazy[b]) -> b, s: b, l: Nel[a]): b = match l {
-        case Nel(x, xs) => f(x, lazy List.foldRightLazy(f, s, xs))
-    }
+    pub def foldRightLazy(f: (a, Lazy[b]) -> b, s: b, l: Nel[a]): b =
```

Comment:
This should be (eventually?) deleted. We will only have one `foldRight`.

Reply:
Yes, I made a `todo`-list in #3226 to make sure I remember to delete the things left over from this PR.

Reply:
👍